cpp/include/cudf/table/table_view.hpp

```diff
@@ -124,7 +124,7 @@ class table_view_base {
    *---------------------------------------------------------------------------**/
   size_type num_rows() const noexcept { return _num_rows; }
 
-  table_view_base() = delete;
+  table_view_base() = default;
```

Comment:
Should this change be pulled somewhere else?

Reply:
Nope - I think this PR is the first time it was needed as a Cython requirement